backend/core/migrations/0100_auto_20220919_0325.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.9 on 2022-09-19 03:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0099_auto_20220830_0405"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="ingredient",
+            name="position",
+            field=models.TextField(),
+        ),
+        migrations.AlterField(
```

Comment:
how does this not explode when we run it?

Reply:
because a numeric field can be converted to a text field

Reply:
ingredients:


```js
> [20,107].sort((a,b) => a - b)
[ 20, 107 ]
> [20,107].sort()
[ 107, 20 ]
> [20,107].map(String).sort()
[ '107', '20' ]
> [20,107].map(String).sort((a, b) => a - b)
[ '20', '107' ]
```